ext/mbstring/php_mbregex.c

```diff
@@ -651,14 +651,16 @@ _php_mb_regex_init_options(const char *parg, size_t narg, OnigOptionType *option
 					*syntax = ONIG_SYNTAX_POSIX_EXTENDED;
 					break;
 				case 'e':
-					if (eval != NULL) *eval = 1;
-					break;
+					zend_value_error("Option \"e\" is not supported");
```

Comment:
You can remove this as a separate case (handle in `default`). Or alternatively, use a different error message like "no longer supported".